docs/user/explanations/aravis-detector.md

```diff
@@ -0,0 +1,102 @@
+# Aravis-detector
+
+This section only covers instructions specific to the Aravis-Detector plugin. For a general overview of Odin Data frameProcessor plugins please refer to the [user](https://odin-detector.github.io/odin-data/master/user/explanations/frame-processor.html) and [developer](https://odin-detector.github.io/odin-data/master/developer/how-to/frame-processor-plugin.html) guides from the odin documentation.
+
+## Plugin Setup
+
+The Aravis-detector should be loaded first in the FrameProcessor plugin chain. It doesn't require the use of the frame receiver app as it directly handles buffers from the aravis camera. To load the plugin you can use the following json command:
+
+```json
+{   "plugin": {
+    "load": {
+        "index": "aravis",
+        "name": "AravisDetectorPlugin",
+        "library": "<Prefix Directory>/lib/libAravisDetectorPlugin.so"
+}}}
+```
+
+:::{Note}
+Here, index refers to the plugin alias for the frame processor and can be treated as a variable name. It will later be used to target the plugin when providing the frame processor with configuration values. The name field must be the same value as the main file name and library field should provide the frame processor with a complete path to the file installed.If you followed the [build](../tutorials/build.md) instructions you should find this in ..../odin_camera_driver/prefix/libraries/.
+:::
+
+The rest of the plugins can then be connected to the aravis-detector. Odin allows plugins to be connected in a tree structure, where plugins can be chained one to another or multiple plugins can be connected into one. For the default aravis plugin config file the live view and hdf plugins are both directly connected to it, similar to plugins 1 and 3 in the following example:
+
+```json
+{   "plugin": {
+        "connect": {
+            "index": "<other plugin 1>",
+            "connection": "aravis"
+}},
+    "plugin": {
+        "connect": {
+            "index": "<other plugin 2>",
+            "connection": "<other plugin 1>"
+}},
+    "plugin": {
+        "connect": {
+            "index": "<other plugin 3>",
+            "connection": "<aravis>"
+}}}
+```
+
+## Plugin Configurations
+
+Different configurations can be passed at start up using the same config file from the setup process. The syntax used is:
+
+```json
+{ "aravis":{
+        "list_devices": true,
+        "compression": "none",
+        "dataset": "data",
+        "ip_address": "169.254.186.47"
+}}
+```
+
+The following list is a complete set of available plugin configuration and their aliases:
+
+| Config | Description| Default value |
+|--------|------------|---------------|
+| ip_address | Connects to the camera with the specified IP address | 127.0.0.1 |
+| exposure_time | Sets the exposure time to the given value | 1000.0 us |
```

Comment:
Question: in ```exposure_time``` and ```frame_rate``` parameters do one need to explicitly set the unit or is the unit fixed and determined on the server so the client just need to set a number? 

For example as a client can I set value to be ```2000.0 us``` or ```2 s``` or just ```2000.0``` (the latter with unit ```us``` implied)?

Reply:
The users supply a numeric value only which the code assumes is in microseconds. Changed the table to better reflect that.